STEERING-COMMITTEE.md

```diff
@@ -90,13 +90,11 @@ first name):
 
 | &nbsp;                                                         | Member           | Organization | Profile                                              |
 | -------------------------------------------------------------- | ---------------- | ------------ | ---------------------------------------------------- |
-| <img width="30px" src="https://github.com/anicolao.png">       | Alex Nicolaou    | Google       | [@anicolao](https://github.com/anicolao)             |
 | <img width="30px" src="https://github.com/thisisnotapril.png"> | April Kyle Nassi | Google       | [@thisisnotapril](https://github.com/thisisnotapril) |
 | <img width="30px" src="https://github.com/bsnchan.png">        | Brenda Chan      | VMware       | [@bsnchan](https://github.com/bsnchan)               |
 | <img width="30px" src="https://github.com/mbehrendt.png">      | Michael Behrendt | IBM          | [@mbehrendt](https://github.com/mbehrendt)           |
 | <img width="30px" src="https://github.com/pmorie.png">         | Paul Morie       | Red Hat      | [@pmorie](https://github.com/pmorie)                 |
-| <img width="30px" src="https://github.com/ronavn.png">         | Ron Avnur        | Google       | [@ronavn](https://github.com/ronavn)                 |
-|                                                                | Open \*          | Google       |                                                      |
+| <img width="30px" src="https://github.com/vaikas.png">         | Ville Aikas      | VMware       | [@vaikas](https://github.com/vaikas)                 |
 
 \* Open seats have been allocated to the organization per the prior committee
```

Comment:
Do we still need this note on open seats?

Reply:
good catch, this should be fixed now :)